Importers/Rhino/Speckle.Importers.Rhino/Internal/FileTypeConfig/ObjConfig.cs

```diff
@@ -0,0 +1,39 @@
+using Rhino;
+using Rhino.FileIO;
+using Speckle.Sdk;
+
+namespace Speckle.Importers.Rhino.Internal.FileTypeConfig;
+
+public sealed class ObjConfig : IFileTypeConfig
+{
+  private readonly FileObjReadOptions _readOptions =
+    new(new FileReadOptions() { OpenMode = true })
+    {
+      MapYtoZ = true,
+      UseObjGroupsAs = FileObjReadOptions.UseObjGsAs.ObjGroupsAsLayers,
+      UseObjObjectsAs = FileObjReadOptions.UseObjOsAs.ObjObjectsAsObjects,
+      IgnoreTextures = true, //We don't support MTL uploads, so no point trying to find a MTL file...
+    };
```

Comment:
Obj import settings means
1. Obj groups become rhino layers
2. We correctly map Y -> Z up
Before:
<img width="1262" height="939" alt="image" src="https://github.com/user-attachments/assets/397980d0-9285-419d-815c-5a7aa70a334f" />

After:
<img width="1230" height="855" alt="image" src="https://github.com/user-attachments/assets/506a8e05-7d40-4413-b613-9bffc2bd77c6" />